File: Models/RoutePricing.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace pathfinderRoute.Models
{
    public class RoutePricing
    {
        public int Id { get; set; }
        public decimal AdditionalCost { get; set; }

        public bool IsFrom { get; set; }
        public bool IsTo { get; set; }
        public int boundaryId { get; set; }
        public Boundary boundary { get; set; }
        public Route route { get; set; }

    }
}

File: ClientApp/src/app/models/routepricing.ts
import { boundary } from "./boundary";

export class routepricing {
    public id: number;
    public additionalCost: number;
    public isFrom: boolean;
    public isTo: boolean;
    public routeId: number;
    public boundaryId: number;
    public boundaryName: string;
    public boundary: boundary;

    public routepricing(){
        this.id = 0;
        this.additionalCost = 0;
        this.isFrom = false;
        this.isTo = false;
        this.boundaryId = 0;
        this.boundary = new boundary();
    }
}